src/server/types.ts

```diff
@@ -19,6 +19,11 @@ export interface RouterOptions {
    *  @default {false}
    */
   trailingSlash?: boolean;
+  /**
+   *  Allows configuring a pattern of files to ignore in islands and routes
+   *  @default {undefined}
```

Comment:
This is `/[\.|_]test\.[t|j]s(x)?$/`, by default. Right? Perhaps, it'd be worth having a short explanation. E.g. By default, ignores all test files with `.test` or `_test` as the filename's suffix.